support/pages/accountPage.ts

```diff
@@ -0,0 +1,31 @@
+import { Locator, Page } from '@playwright/test'
+
+export class AccountPage {
+    readonly page: Page
+    readonly accountMenuBtn: Locator
+    readonly accountManageBtn: Locator
+    readonly progressBarSelector: Locator
+    readonly progressBarCurrent: Locator
+    readonly progressBarOption: Locator
+
+    constructor(page: Page) {
+        this.page = page
+        this.accountMenuBtn = this.page.locator('.oc-topbar-avatar')
+        this.accountManageBtn = this.page.locator('#oc-topbar-account-manage')
+
+        this.progressBarSelector = this.page.locator('.extension-preference .vs__search')
+        this.progressBarCurrent = this.page.locator('.extension-preference .vs__selected span')
+        this.progressBarOption = this.page.getByText('Nyan Cat progress bar')
+    }
+
+    async goToAccountPage() {
+        await this.accountMenuBtn.click()
+        await this.accountManageBtn.click()
+    }
+
+    async selectProgressBarExtension(){
+        await this.progressBarSelector.waitFor()
+        await this.progressBarSelector.click()
```

Comment:
IMO click wait for selector to be visibile https://playwright.dev/docs/api/class-locator#locator-click why to add expicit waitfor ? 
```suggestion
        await this.progressBarSelector.click()
```

Reply:
Yeah! You are right; it looks unnecessary. 🚀 i will remove it